x-pack/plugins/synthetics/public/apps/synthetics/components/settings/global_params/delete_param.tsx

```diff
@@ -35,14 +35,14 @@ export const DeleteParam = ({
 
   const { status } = useFetcher(() => {
     if (isDeleting) {
-      return deleteGlobalParams({ ids: items.map(({ id }) => id) });
+      return deleteGlobalParams(items.map(({ id }) => id));
     }
   }, [items, isDeleting]);
 
   const name = items
     .map(({ key }) => key)
     .join(', ')
-    .substr(0, 50);
+    .slice(0, 50);
```

Comment:
The only reason for this is that `substr` is deprecated and we can use `slice` as a drop-in replacement.